examples/streamServer.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+use Legionth\React\Http\HttpServer;
+use Legionth\React\Http\HttpBodyStream;
+use React\Socket\Server as Socket;
+use RingCentral\Psr7\Response;
+use Psr\Http\Message\RequestInterface;
+use React\Stream\ReadableStream;
+
+require __DIR__ . '/../vendor/autoload.php';
+
+
+$loop = React\EventLoop\Factory::create();
+
+$callback = function(RequestInterface $request) use ($loop){
+    $input = new ReadableStream();
+
+    $periodicTimer = $loop->addPeriodicTimer(0.5, function () use ($input) {
+        $input->emit('data', array("6\r\nworld\n\r\n"));
```

Comment:
There's no documentation on what this line is supposed to to. Besides, does this really make sense to ask consumers of this lib to encode this themselves?